app/models/content_item.rb

```diff
@@ -217,10 +225,14 @@ def should_register_routes?(previous_item: nil)
     true
   end
 
-  def authorised_user_uids
+  def auth_user_ids
     access_limited.fetch('users', [])
   end
 
+  def auth_org_ids
```

Comment:
I don't think you introduced this, so apologies. But I think it's premature to abbreviate this to org

Reply:
Changed back to `organisation`